### ISSUE TITLE
Addition of functionality

### ISSUE THREAD

**Issue by z0zz0** [NieuwlandGeo/SLDReader#116]
Hi,

I was in touch with Arjen through mail asking about a VendorOption. He told me to create a Github issue including a full exmple of a SLD.

**My email:**
I have a specific question about SLDReader capabilities, namely regarding a VendorOption.
```
<VendorOption name="placement">lastPoint</VendorOption>
<VendorOption name="placement">firstPoint</VendorOption>
```
This works fine in QGIS but in our web application, where we use SLDReader it is not looking fine, see two pictures below for example. I was wondering if it is possible to have SLDReader also support the VendorOption used above? 

**Arjen's answer:**
I think it should be possible to add this functionality, but probably not soon.
Can you create an issue in Github including a full example of an SLD that uses this placement?

### LinearAnnotation. 
**As it ought to be shown:**
![arrow](https://user-images.githubusercontent.com/44770272/155144714-6f509c08-b53e-4096-9654-932b4fb8dd55.PNG)

**As it is shown:**
![faulty arrow](https://user-images.githubusercontent.com/44770272/155145070-68a54d48-9018-4488-a2f5-3dfec6caaeb0.PNG)

### LinearDimension. 
**As it ought to be shown:**
![bi-directional arrow](https://user-images.githubusercontent.com/44770272/155144892-3b93143f-59d5-4051-98db-367c186b4be4.PNG)

**As it is shown:**
![faulty bi-directional arrow](https://user-images.githubusercontent.com/44770272/155145212-8e37d7f3-0675-4773-93a1-68ada20d36cd.PNG)

### SLD Example
[styles.txt](https://github.com/NieuwlandGeo/SLDReader/files/8117155/styles.txt)

**Comment by ajkopinga** [comment]
I have published a new v0.2.14 release that implements these two vendor options.

**Comment by z0zz0** [comment]
Many thanks, cheers! :)
